aten/src/ATen/native/mps/operations/RnnOps.mm

```diff
@@ -356,6 +356,10 @@
                                                                                bool bidirectional,
                                                                                bool batch_first) {
   using namespace mps;
+  TORCH_CHECK(num_layers <= 2,
+              "MPS LSTM gradient kernel has an issue with the gradients for num_layers > 2. ",
+              "As a temporary fix, you can set the environment variable `PYTORCH_ENABLE_MPS_FALLBACK=1` ");
```

Comment:
I think that environment variable doesn't work as the LSTM kernel for mps is already implemented? Maybe do a `TORCH_WARN_ONCE` and redispatch to cpu?